storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -6945,6 +6973,12 @@ bool xb_init() {
       return (false);
     }
 
+    if (opt_page_tracking &&
+        !xb_page_tracking_component_installed(mysql_connection)) {
+      const char *query = "INSTALL COMPONENT \"file://component_mysqlbackup\"";
+      xb_mysql_query(mysql_connection, query, true, false);
```

Comment:
```suggestion
      xb_mysql_query(mysql_connection, query, false, true);
```

Do we need the result set?
Should we abort on error ?

Reply:
fixed. yeah. aborting on error make sense.